examples/erc1155/src/lib.rs

```diff
@@ -47,46 +50,69 @@ impl Erc1155Example {
         amounts: Vec<U256>,
         data: Bytes,
     ) -> Result<(), Vec<u8>> {
+        self.pausable.when_not_paused()?;
         self.erc1155._mint_batch(to, token_ids, amounts, &data)?;
         Ok(())
     }
 
-    pub fn set_operator_approvals(
-        &mut self,
-        owner: Address,
-        operator: Address,
-        approved: bool,
-    ) -> Result<(), Vec<u8>> {
-        self.erc1155
-            ._operator_approvals
-            .setter(owner)
-            .setter(operator)
-            .set(approved);
-        Ok(())
-    }
-
-    fn burn(
+    pub fn burn(
```

Comment:
Is the `pub` keyword necessary? The tests run just fine without it

Reply:
Not needed tbh.

Reply:
But each of our callable fn is `pub` so just for consistency.